plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergOrcFileWriter.java

```diff
@@ -157,6 +157,12 @@ private static Metrics computeMetrics(MetricsConfig metricsConfig, Schema iceber
                 upperBounds.isEmpty() ? null : upperBounds);
     }
 
+    @Override
+    public long getBufferedBytes()
+    {
+        return 0; //buffered bytes are already counted in written bytes by orcFileWriter
```

Comment:
Sounds like undesired inconsistency. Would it be possible to fix it instead?

Reply:
I will try to that in a separate PR after this one, ok ?

Reply:
This might be as simple as adding `parquetWriter.getBufferedBytes()` at
https://github.com/trinodb/trino/blob/9ab09e99d6c56fe955697a51a95400c367525133/plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetFileWriter.java#L89

as a followup we may want to document or rename the method.

i would prefer to fix the inconsistency rather than introduce a method, if the end result is that the method is going to be removed soonish

Reply:
yes it may be that simple but it will change behaviour of `getWrittenBytes` and I am not sure what depends on that. For example `SortingFileWritter` depends on that.. I'd prefer to do it as a separate PR to be easily able to see what is broken(if anything) by this change.

@findepi  Do you agree?

Reply:
> yes it may be that simple but it will change behaviour of `getWrittenBytes` and I am not sure what depends on that.

For Parquet -- yes
for ORC -- no

> SortingFileWritter depends on that.. 

if this is a problem, we already have a problem (for ORC), right?